plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/PostgresResources.properties

```diff
@@ -172,4 +172,13 @@ dialog_trigger_label_combo_event_type_tip = Choose trigger event type
 dialog_trigger_label_title = Create new Event Trigger
 
 dialog_trigger_edit_page_label_trigger_function = Trigger function
-dialog_trigger_edit_page_select_function_title = Select function for trigger
\ No newline at end of file
+dialog_trigger_edit_page_select_function_title = Select function for trigger
+
+procedure_check_label = Check
+procedure_check_label2 = "{0}" check
```

Comment:
The `"Something" check` label looks weird to me. Wouldn't it be more natural if spelled `Check "Something"` instead?